src/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ if(${NUPIC_IWYU})
   endif()
 endif()
 
+option(NUPIC_INSTALL_INCLUDES "Force installation of include files")
```

Comment:
I wouldn't say "force" since that implies overriding some other behavior. Something like "Toggle whether to install headers, libraries, and executables to the install location for C++ clients" might be better